src/components/Exchange/SwapBox.js

```diff
@@ -1786,6 +1786,18 @@ export default function SwapBox(props) {
     }
   }
 
+  function getFundingFee() {
```

Comment:
would it be better to name it `renderFundingFee`? i think other functions that return react elements use that convention. so with `render` prefix we'll know what it does

Reply:
Updated!